File: agent/bindings/csharp/CodyContextFilterItem.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class CodyContextFilterItem
  {
    [JsonProperty(PropertyName = "repoNamePattern")]
    public string RepoNamePattern { get; set; }
    [JsonProperty(PropertyName = "filePathPatterns")]
    public string[] FilePathPatterns { get; set; }
  }
}

File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/CodyContextFilterItem.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class CodyContextFilterItem(
  val repoNamePattern: String,
  val filePathPatterns: List<String>? = null,
)

